graph.py

```diff
@@ -0,0 +1,55 @@
+from collections import defaultdict
+
+class Graph:
+    def __init__(self) -> None:
+        self.adjacency_list = defaultdict(list)
+    
+    def add_edge(self, u, v):
+        self.adjacency_list[u].append(v)
+        self.adjacency_list[v].append(u)
```

Comment:
Can your graph's implementation handle directed edges??

Reply:
Yes. 
while adding edge 
if you say add_edge(u,v) it indicates edge from u to v
If you sat add_edge(v,u) it indicates edge from v to u

Reply:
If we make this as directed graph, undirected graph is extending this with one more step, which implementation best suits???

Reply:
done